lib/input-view.js

```diff
@@ -11,11 +16,16 @@ export default class InputView {
   miniEditor: atom$TextEditor;
   panel: ?atom$Panel;
   previouslyFocusedElement: ?HTMLElement;
-  constructor({ prompt, defaultText, allowCancel }: opts, onConfirmed: cb) {
+  constructor(
+    { prompt, defaultText, allowCancel, password }: opts,
+    onConfirmed: cb
+  ) {
     this.onConfirmed = onConfirmed;
 
     this.element = document.createElement("div");
     this.element.classList.add("hydrogen", "input-view");
+    if (password) this.element.classList.add("password");
```

Comment:
We could also add the CSS directly to the scroll view, but this should work too.